package.json

```diff
@@ -3,13 +3,21 @@
   "version": "0.1.0",
   "devDependencies": {
     "grunt": "~0.4.1",
+    "grunt-closure-linter": "~0.1.0",
```

Comment:
It's probably not related to migration to karma but will just leave it here..

Not sure if this dependency is still viable, assuming that closure linter is deprecated.

Per https://developers.google.com/closure/utilities we should rather compile with --jscomp_warning=lintChecks

Reply:
Oops, not sure how that got there